### ISSUE TITLE
How to not have today selected by default?

### ISSUE THREAD

**Issue by SachaG** [react-component/calendar#259]
Is there a way to not have the calendar highlight the current day as selected (with `.rc-calendar-selected-day`) by default? I would like the default state to have no day at all selected.

(And by the way, great work on porting this library to Moment!)

**Comment by paranoidjk** [comment]
just use props.defaultValue，please at least read the api doc in the README before you file a issue

**Comment by paranoidjk** [comment]
by the way，can you tell us why you in favor of momentjs inside？we are consider to remove momentjs dependence. https://github.com/ant-design/ant-design-mobile/issues/1217

**Comment by SachaG** [comment]
I tried that:

```
<Calendar
          defaultValue={null}
        />
```

The current date still appears selected.

**Comment by benjycui** [comment]
> The current date still appears selected.

Yep, we had noticed this issue, trace: https://github.com/ant-design/ant-design/issues/6146

**Comment by SachaG** [comment]
> can you tell us why you in favor of momentjs inside？

I meant compared to the previous dependency on the Gregorian Calendar package. At least Moment is a pretty standard package and I was already using it anyway in my codebase.
